server/Dockerfile

```diff
@@ -4,7 +4,6 @@ WORKDIR /usr/src/app
 # Install dependencies
 RUN apt-get update && \
     apt-get install -y \
-    make \
     dos2unix \
     libmagic1
 
```

Comment:
if we don't build the react app anymore we can also skip `COPY ../web web`

Reply:
removed